src/OrchardCore/OrchardCore.ContentManagement.Abstractions/IContentManager.cs

```diff
@@ -38,7 +38,7 @@ public interface IContentManager
     /// if an existing draft is present.</item>
     /// </list>
     /// </remarks>
-    Task CreateAsync(ContentItem contentItem, VersionOptions options = null);
+    Task<bool> CreateAsync(ContentItem contentItem, VersionOptions options = null);
```

Comment:
We should avoid this breaking change. Consider adding a new method, e.g. `TryCreateAsync` and add a default interface implementation for that.

Reply:
We did something like that in https://github.com/OrchardCMS/OrchardCore/pull/17441